src/PrestaShopBundle/Resources/config/services/adapter/attribute.yml

```
@@ -1,6 +1,7 @@
 services:
   _defaults:
-    public: true
+    public: false
+    autowire: true
```

```suggestion
    autowire: true
    autoconfigure: true
```
We could maybe add this globally here, since most of the services already rely on it (and the ones that don't shouldn't be impacted anyway)